src/index.js

```diff
@@ -9,5 +9,39 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+$( document ).ready(function() {
 
-console.log('This is the JavaScript entry file - your code begins here.');
+$('.login-button').on( "click", function() {
+
+  let userName = $('.user-login').val();
+  let userpassword = $('.password').val();
+
+  if (userName === 'manager' && userpassword === 'overlook2019') {
+    window.location = '/src/manager.html';
```

Comment:
Try putting a . in front of your file path, like this: './src/manager.html'

Reply:
i tried that i got error Cannot GET /.src/manager.html

Reply:
Oo, I've ran into that problem, too.  I believe that file path that's displayed as an error is the filepath you have in your webpack.  So, go in there and make sure you have the . in the correct place, too.